scour/scour.py

```diff
@@ -3985,6 +3997,10 @@ def format_usage(self, usage):
 _option_group_optimization.add_option("--disable-style-to-xml",
                                       action="store_false", dest="style_to_xml", default=True,
                                       help="won't convert styles into XML attributes")
+_option_group_optimization.add_option("--style",
+                                      action="store", type="string", dest="style_type", default="none", metavar="TYPE",
+                                      help="style type (overrides style-to-xml): none, preserve, inline,"
```

Comment:
Should we call it `inline-css` rather than `inline` to be more explicit?  At least, I did not connect `inline` with an inline CSS attribute at first.